src/main/java/com/dataloom/neuron/audit/AuditLogQueryService.java

```diff
@@ -24,23 +44,25 @@
     private final Session           session;
     private final PreparedStatement storeQuery;
 
-    public AuditLogQueryService( String keyspace, Session session ) {
+    // TODO: what would CassandraConfiguration be needed for in the future?
+    public AuditLogQueryService( CassandraConfiguration cassandraConfig, Session session ) {
 
         this.session = session;
         this.storeQuery = session.prepare( Table.AUDIT_LOG.getBuilder().buildStoreQuery() );
     }
 
     public void store( AuditableSignal signal ) {
 
+        // TODO: how do I deal with Optionals?
```

Comment:
In the case that an event is not related objects, like "You've successfully verified your email!".